app/src/main/java/org/coepi/android/MainActivity.kt

```diff
@@ -49,7 +55,7 @@ class MainActivity : AppCompatActivity() {
         setTheme(AppTheme)
         super.onCreate(savedInstanceState)
         setContentView(activity_main)
-
+        setReminderAlarmsFor(14)
```

Comment:
Can you double-check if this doesn't create a new alarm every time when you start the app? Maybe AlarmManager is smart enough.

Reply:
Should work as intended:
` If there is already an alarm for this Intent
     * scheduled (with the equality of two intents being defined by
     * {@link Intent#filterEquals}), then it will be removed and replaced by
     * this one.`